GithubBrowserSample/app/src/main/res/layout/repo_fragment.xml

```diff
@@ -24,17 +24,17 @@
 
         <import type="com.android.example.github.vo.Resource" />
 
-        <variable
-            name="repo"
-            type="com.android.example.github.vo.Repo" />
+        <import type="androidx.lifecycle.LiveData"/>
```

Comment:
This is strange. Normally you'd use LiveData directly without wrapping the type. Can you add a comment explaining why you're doing this?

Reply:
not sure what you mean here. it is just and import and then we specify type in the varaible.